ctlra/midi.c

```diff
@@ -16,6 +16,9 @@ struct ctlra_midi_t {
 	void *input_cb_ud;
 };
 
+// Maximum size of (sysex) message we expect to deal with on input.
+#define MAX_MSG_SIZE 1024
+
```

Comment:
I believe these changes are similar to those in Alberts branch, which are going to be merged soon into master (https://github.com/openAVproductions/openAV-Ctlra/pull/109#issuecomment-1352343199)